src/plugins/point_in_time_management/public/components/pit_table/pit_table.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState } from 'react';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { useEffectOnce, useMount } from 'react-use';
+import {
+  EuiButton,
+  EuiInMemoryTable,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiSpacer,
+  EuiSearchBarProps,
+} from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { FormattedMessage } from '@osd/i18n/react';
+import { getListBreadcrumbs } from '../breadcrumbs';
+import { PointInTimeManagementContext } from '../../types';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { getDataSources } from '../utils';
+import { EmptyState } from './empty_state';
+import { PageHeader } from './page_header';
+
+export interface DataSourceItem {
+  id: string;
+  title: string;
+  sort: string;
+}
+
+const PITTable = ({ history }: RouteComponentProps) => {

```

Comment:
Can we please add unit tests for this component ?

Reply:
Will add tests in the following PR when we add logic to fetch the PITs

Reply:
Yes, pls add more unit tests.